tutorials/fundamentals/installation.md

```diff
@@ -1,30 +1,47 @@
-# napari installation tutorial
+# How to install napari on your machine
 
-Welcome to the **napari** installation tutorial!
+Welcome to the **napari** installation guide!
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
```

Comment:
I think we may want to keep this redirection for people looking for the developer installation guide instead

Reply:
Ah, that link doesn't work anyway. Here it is: https://napari.org/developers/contributing.html

Reply:
Yep, I've added this at the very top in a note:

```{note} 
If you want to contribute back to the napari codebase, you should follow [this guide instead](https://napari.org/developers/contributing.html).
```